easybuild/tools/build_log.py

```diff
@@ -58,6 +59,10 @@
 DRY_RUN_MODULES_INSTALL_DIR = None
 
 
+DEVEL_LOG_LEVEL = logging.DEBUG - 1
```

Comment:
no, it's a lower logging level; `log.devel` messages should not be included when using `--debug`, but `log.debug` messages *should* be included when using `--devel` (see also the added test)